http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2912,6 +2910,16 @@ components:
         cloudService:
           type: string
           enum: [gce, dataproc]
+    OneOfRuntimeConfig
+      oneOf:
+        - $ref: "#/components/schemas/GceConfig"
+        - $ref: "#/components/schemas/GceWithPdConfig"
+        - $ref: "#/components/schemas/MachineConfig"
+      discriminator:
+        propertyName: cloudService
+        mapping:
```

Comment:
This corrects the usage of the discriminator field: https://swagger.io/specification/#discriminator-object